ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4259,6 +4265,9 @@ CHWSWISOVM:
   guid: "1a187792-767f-4132-9333-161c6db07ce4"
   description: "Supply side isolation valve monitoring."
   is_abstract: true
+  opt_uses:
```

Comment:
Should these instead be `failed_chilled_supply_water_isolation_valve_alarm` and `local_override_chilled_supply_water_isolation_valve_alarm`?

Reply:
Fair point, fixed.

Reply:
Can remove these because you are not using chilled water anymore.